test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3138,6 +3122,35 @@ def forward(self, x):
         x = torch.randn(2, 5)
         self.run_test(ShapeModule(), (x,), rtol=1e-3, atol=1e-5)
 
+    @skipIfUnsupportedMinOpsetVersion(12)
+    def test_celu(self):
+        class Celu(torch.nn.Module):
+            def __init__(self):
+                super(Celu, self).__init__()
+                self.celu = torch.nn.CELU(alpha=1.0)
```

Comment:
add one test case with a default alpha and one test case with an alpha with a different value than the default value (which is 1) ?